hack/refresh-examples.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/env bash
+# Copyright 2022 Red Hat, Inc.
```

Comment:
Can we just use `Copyright The Enterprise Contract Contributors` like we do in ec-cli?

Reply:
Tots, not sure where I got this from...

Reply:
Actually all scripts in `hacks` are doing Red Hat, Inc... I'll do that in a separate PR...